src/styles/main.scss

```diff
@@ -1,3 +1,83 @@
+@import "./variables.scss";
+
 body {
   margin: 0;
 }
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  gap: 1px;
+  margin-top: 100px;
+  margin-left: $general-margin;
+  margin-right: $general-margin;
+  text-align: center;
+  
+  &__elem {
+    display: block;
+    width: 98px;
+    height: 98px;
```

Comment:
Think about changing box-sizing, so you don't need to set sizes without borders. Also, would be great to create a variable for elem size.
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
last home work i had changed box-sizing values and i had tip from mentor that it box-sizing better to set value 'content-box' ofcourse i will change if it is not correct but, could you explaine to me how it s correct in generaly ?

Reply:
`box-sizing: border-box` is better for this task. In this way, you will not need to take into account the size of the borders and calculate them yourself from the size of the content.
Regarding the previous case, if you do not understand, then ask in the fe_chat and tag the mentor so that they explain to you why this particular solution is better for that task